app/jobs/record_submitted_claim_job.rb

```diff
@@ -0,0 +1,7 @@
+require "geckoboard"
+
+class RecordSubmittedClaimJob < ApplicationJob
+  def perform(claim)
+    Claim::GeckoboardEvent.new(claim, :submitted, claim.submitted_at).record
```

Comment:
This reads much nice than `RecordClaimEvent.new(...).perform` we had before 👍

Reply:
👍